templates/etc/clamd.d/scan.conf.j2

```diff
@@ -99,15 +104,23 @@ LogSyslog yes
 
 # TCP port address.
 # Default: no
+{% if CLAMAV_SOCKET_TYPE == 'file' %}
+#TCPSocket 3310
+{% else %}
 TCPSocket 3310
+{% endif %}
 
 # TCP address.
 # By default we bind to INADDR_ANY, probably not wise.
 # Enable the following to provide some degree of protection
 # from the outside world. This option can be specified multiple
 # times if you want to listen on multiple IPs. IPv6 is now supported.
 # Default: no
+{% if CLAMAV_SOCKET_TYPE == 'file' %}
+#TCPAddr 127.0.0.1
```

Comment:
Can the tcp bind address and port be converted to vars? There are environments where we need to have clamav listening on a different port, due to 3310 being used by other service.